Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/01-prerequisites.md

```diff
@@ -8,6 +8,7 @@
         - Contributor role is required at the subscription level to have the ability to create resource groups and perform deployments.
         - User Access Administrator role is required at the subscription level since you'll be performing role assignments to managed identities across various resource groups.
         - Global Admin on Azure AD Tenant is required for setting up Azure Application Proxy. This setup is done manually. An admin could perform this step for you as it's the last step in the setup after deploying your application. 
+    Please follow [these instructions](https://learn.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal) to create a service principal in Azure.
```

Comment:
Do you need to assign RBAC to it? (all of the above roles?)  If so, please be clear about that here.